bee-storage/bee-storage-sled/src/config.rs

```diff
@@ -105,34 +89,34 @@ impl From<SledConfigBuilder> for SledConfig {
     }
 }
 
-/// Configuration related to the access operations of the storage.
-#[derive(Clone)]
-pub struct StorageConfig {
-    pub(crate) fetch_edge_limit: usize,
-    pub(crate) fetch_index_limit: usize,
-    pub(crate) fetch_output_id_limit: usize,
-}
-
-/// Configuration builder related to the access operations of the storage.
+/// Builder for an [`AccessConfig`].
 #[derive(Default, Deserialize)]
-pub struct StorageConfigBuilder {
-    fetch_edge_limit: Option<usize>,
-    fetch_index_limit: Option<usize>,
-    fetch_output_id_limit: Option<usize>,
-}
+pub struct AccessConfigBuilder {}
 
-impl StorageConfigBuilder {
+impl AccessConfigBuilder {
```

Comment:
What is the purpose of this builder here, if `AccessConfig` is a unit struct?

Reply:
It is temporarily a unit struct, it's just that we don't have the types that require it to have fields in the coordicide branch yet

Reply:
Ah, makes sense!